<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Apprendre</class>
 <widget class="QDialog" name="Apprendre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Module d'apprentissage</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>141</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>540</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>540</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>-20</y>
     <width>621</width>
     <height>311</height>
    </rect>
   </property>
   <property name="text">
    <string>sch√©ma circuit</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>371</width>
     <height>53</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>371</width>
     <height>53</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget_4">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>820</y>
     <width>651</width>
     <height>51</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <row>
    <property name="text">
     <string>valeur</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string notr="true">opcode</string>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>accZ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc15</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selB</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selC</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>pc_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ir_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc_oe</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>alufs</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>rnw</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>700</y>
     <width>651</width>
     <height>51</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <row>
    <property name="text">
     <string>valeur</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string notr="true">opcode</string>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>accZ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc15</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selB</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selC</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>pc_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ir_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc_oe</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>alufs</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>rnw</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>760</y>
     <width>651</width>
     <height>51</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <row>
    <property name="text">
     <string>valeur</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string notr="true">opcode</string>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>accZ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc15</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selB</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selC</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>pc_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ir_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc_oe</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>alufs</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>rnw</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>290</y>
     <width>651</width>
     <height>51</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <row>
    <property name="text">
     <string>valeur</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string notr="true">opcode</string>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>accZ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc15</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selB</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selC</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>pc_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ir_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc_oe</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>alufs</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>rnw</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget_6">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>470</y>
     <width>651</width>
     <height>51</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <row>
    <property name="text">
     <string>valeur</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string notr="true">opcode</string>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>accZ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc15</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selB</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selC</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>pc_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ir_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc_oe</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>alufs</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>rnw</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget_7">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>350</y>
     <width>651</width>
     <height>51</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <row>
    <property name="text">
     <string>valeur</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string notr="true">opcode</string>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>accZ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc15</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selB</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selC</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>pc_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ir_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc_oe</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>alufs</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>rnw</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidget_8">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>410</y>
     <width>651</width>
     <height>51</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <row>
    <property name="text">
     <string>valeur</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string notr="true">opcode</string>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>accZ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc15</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selA</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selB</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>selC</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>pc_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ir_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc_ld</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>acc_oe</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>alufs</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>rnw</string>
    </property>
   </column>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>listWidget</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit</zorder>
  <zorder>tableWidget_4</zorder>
  <zorder>tableWidget</zorder>
  <zorder>tableWidget_2</zorder>
  <zorder>tableWidget_5</zorder>
  <zorder>tableWidget_6</zorder>
  <zorder>tableWidget_7</zorder>
  <zorder>tableWidget_8</zorder>
 </widget>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>